comit/src/network/protocols/orderbook.rs

```diff
@@ -118,7 +119,7 @@ impl Orderbook {
     pub fn peer_id_for_order(&self, order_id: OrderId) -> Option<PeerId> {
```

Comment:
This method should maybe even actually return the `MakerId` instead of the `PeerId`. Would make it more descriptive that we are actually getting the maker of the order here.

Reply:
The fn should probably also be private :)

Reply:
Good ideas, implemented as suggested.